data/categories.txt

```diff
@@ -4763,6 +4763,318 @@ vi:Nhà vật lý trị liệu
+cs:Optometrie|Optometrista
+da:Optometri|Optometrist
+de:Optometrie|
+el:Οπτομετρία|Οπτομέτρης
+es:Optometría|Optometrista
+et:Optomeetria|Optometrist
+eu:Optometria|Optometrista
+fa:بینایی سنجی|اپتومتریست
+fi:Optometria|Optometristi
+fr:Optométrie|Optométriste
+he:אופטומטריה|אופטומטריסט
+hu:Optometria|Optometrista
+id:Optometri|Ahli kacamata
+it:Optometria|Optometrista
+ja:オプトメトリー|オプトメトリスト
+ko:검안|검안사
+mr:ऑप्टोमेट्री|ऑप्टोमेट्रिस्ट
+nb:Optometri|Optometrist
+nl:Optometrie|Optometrist
+pl:Optometria|Optometrysta
+pt:Optometria|Optometrista
+pt-BR:Optometria|Optometrista
+ro:Optometrie|Optometrist
+ru:Оптометрия|Оптометрист
+sk:Optometria|Optometrista
+sv:Optometri|Optometrist
+sw:Optometry|Daktari wa macho
+th:ทัศนมาตรศาสตร์|นักทัศนมาตร
+tr:Optometri|Optometrist
+uk:Оптометрія|Оптометрист
+vi:Khúc xạ nhãn khoa|bác sĩ nhãn khoa
+zh-Hans:验光配镜|验光师
+zh-Hant:驗光|驗光師
+
+healthcare-podiatrist
+en:Podiatry|Podiatrist
+ar:علاج الأرجل|طبيب الأرجل
+be:Подологія|Артапед
+bg:Подиология|Подиатрист
+cs:Podiatrie|Podiatr
+da:Fodterapi|Fodterapeut
+de:Podologie|Podologe|Podologin
+el:Ποδιατρική|Ποδίατρος
+es:Podología|Podólogo
+et:Jalaravi
+eu:Podologia|Podologoa
+fa:پزشکی پا|متخصص پا
+fi:Jalkaterapia|Jalkaterapeutti
+fr:Podologie|Podologue
+he:פודיאטריה|פודיאטר
+hu:Podiatria|Podiatrist
+id:Podiatri|Ahli penyakit kaki
+it:Podologia|Podologo
+ja:足病学|足病医
+ko:발병학|발 전문의
+mr:पोडियाट्री|पोडियाट्रिस्ट
+nb:Fotterapi|Fotterapeut
+nl:Podologie|Podotherapeut
+pl:Podiatria|Podolog
+pt:Podiatria|Podiatra
+pt-BR:Podiatria|Podiatra
+ro:Podiatrie|Podiatru
+ru:Подиатрия|Ортопед
+sk:Podiatria|Podológ
+sv:Fotvård|Fotvårdare
+sw:Matibabu ya miguu|Daktari wa miguu
+th:แก้โรคเท้า|แพทย์โรคเท้า
+tr:Podiatri|Podiatrist
+uk:Подологія
+vi:khoa bàn chân|Bác sĩ chân
+zh-Hans:足科|足科医生
+zh-Hant:足病學|足病醫生
+
+healthcare-psychotherapist
+en:Psychotherapy
+ar:العلاج النفسي|معالج نفسي
+be:Псіхатэрапія|Псіхатэрапеўт
+bg:Психотерапия|Психотерапевт
+cs:Psychoterapie|Psychoterapeut
+da:Psykoterapi|Psykoterapeut
+de:Psychotherapie|Psychotherapeutin|Psychotherapeut
+el:Ψυχοθεραπεία|Ψυχοθεραπευτής
+es:Psicoterapia|Psicoterapeuta
+et:Psühhoteraapia|Psühhoterapeut
+eu:Psikoterapia|Psikoterapeuta
+fa:روان درمانی|روان درمانگر
+fi:Psykoterapia|Psykoterapeutti
+fr:Psychothérapie|Psychothérapeute
+he:פסיכותרפיה|פסיכותרפיסט
+hu:Pszichoterápia|Pszichoterapeuta
+id:Psikoterapi|Psikoterapis
+it:Psicoterapia|Psicoterapeuta
+ja:心理療法|サイコセラピスト
+ko:심리치료|심리 치료사
+mr:मानसोपचार|मानसोपचारतज्ज्ञ
+nb:Psykoterapi|Psykoterapeut
+nl:Psychotherapie|Psychotherapeut
+pl:Psychoterapia|Psychoterapeuta
+pt:Psicoterapia|Psicoterapeuta
+pt-BR:Psicoterapia|Psicoterapeuta
+ro:Psihoterapie|Psihoterapeut
+ru:Психотерапия|Психотерапевт
+sk:Psychoterapia|Psychoterapeut
+sv:Psykoterapi|Psykoterapeut
+sw:Tiba ya kisaikolojia|Mwanasaikolojia
+th:จิตบำบัด|นักจิตบำบัด
+tr:Psikoterapi|Psikoterapist
+uk:Психотерапія|Психотерапевт
+vi:Tâm lý trị liệu|nhà trị liệu tâm lý
+zh-Hans:心理治疗|心理治疗师
+zh-Hant:心理治療|心理治療師
+
+healthcare-sample_collection
```

Comment:
Please, comment/remove this category.
Due to search specific it will interfere with other categories and translations are strange IMHO (because of "sample").